tests/fixture/tmpnet/flags/runtime.go

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package flags
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/spf13/pflag"
+
+	"github.com/ava-labs/avalanchego/tests/fixture/tmpnet"
+)
+
+var validRuntimes = []string{
```

Comment:
nit you could use `...` to make it an array so it's a bit less mutable. I'm paranoid and usually use a function `func validRuntimes() []string` to not have a global variable that could be mutated, but I would not recommend that necessarily in this case :smile: 
I suppose one of my unpopular opinion is zero global variable except for necessary reasons, see [my little (dusty old) blog post](https://qqq.ninja/blog/post/go-global-variables/) (I would had embed variables which are necessary too). Again, very unpopular opinion 😄 🤷 

```suggestion
var validRuntimes = [...]string{
```